internal/jobcontainers/path_test.go

```diff
@@ -30,6 +35,11 @@ func TestGetApplicationName(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	_, _, err = getApplicationName("./ping", cwd, os.Getenv("PATH"))
```

Comment:
Why not test what the `path` and `commandLine` variables are here?  Is not getting an error enough?

Reply:
This is mostly just testing that we found the application, so we resolved to C:\windows\system32\ping.exe, I can add an explicit check to make sure this is true however